### PR TITLE
fix(bbb-conf): bbb-conf tomcat systemctl

### PR DESCRIPTION
**Pull request by BrentBaccala**
Only attempt TOMCAT_SERVICE systemctl operations if TOMCAT_SERVICE exists

It doesn't make sense (to me) that these systemctl operations are outside of the if blocks.

When I installed a 2.6.0-alpha.2 server, I got these messages during the install:

```
Starting BigBlueButton
Too few arguments.

# Warning:  could not be started. Please, check BBB-LTI.
#     Run the command:
#       sudo journalctl -u 
#     To better understand the ERROR

```

### CODE REVIEW

**Comment by antobinary** [comment]
Thanks @BrentBaccala 
This seems to do similar changes to https://github.com/bigbluebutton/bigbluebutton/pull/15307

P.S. I tweaked the destination branch to be v2.6.x-release

**Comment by GuiLeme** [comment]
Hey @BrentBaccala, I suggested a slight change in the code just to ensure those warnings won't break and display incomplete information.

**Review by GuiLeme — CHANGES_REQUESTED** [review]

File: bigbluebutton-config/bin/bbb-conf [hunk 1]
```diff
@@ -377,15 +376,15 @@ start_bigbluebutton () {
...
-    fi
 
-    systemctl start $TOMCAT_SERVICE || {
-        echo
-        echo "# Warning: $TOMCAT_SERVICE could not be started. Please, check BBB-LTI."
-        echo "#     Run the command:"
-        echo "#       sudo journalctl -u $TOMCAT_SERVICE"
-        echo "#     To better understand the ERROR"
-    }
+        systemctl start $TOMCAT_SERVICE || {
```
[the remark this answers is not in the document]
Comment (antobinary) [reply bigbluebutton-config/bin/bbb-conf:380]: This change has been used by others with success https://github.com/bigbluebutton/bigbluebutton/issues/15387#issuecomment-1185419014

File: bigbluebutton-config/bin/bbb-conf [hunk 2]
```diff
@@ -346,9 +346,8 @@ stop_bigbluebutton () {
 
     if systemctl list-units --full -all | grep -q $TOMCAT_USER.service; then
       TOMCAT_SERVICE=$TOMCAT_USER
+      systemctl stop $TOMCAT_SERVICE bigbluebutton.target
```
Comment (antobinary) [inline bigbluebutton-config/bin/bbb-conf:349]:
```suggestion
      systemctl stop $TOMCAT_SERVICE
```

File: bigbluebutton-config/bin/bbb-conf [hunk 3]
```diff
@@ -346,9 +346,8 @@ stop_bigbluebutton () {
...
     fi
-
-    systemctl stop $TOMCAT_SERVICE bigbluebutton.target
```
Comment (antobinary) [inline bigbluebutton-config/bin/bbb-conf:351]: systemctl stop bigbluebutton.target

Comment (antobinary) [reply bigbluebutton-config/bin/bbb-conf:351]: (otherwise we won't stop BBB if TOmcat is not present)

**Comment by antobinary** [comment]
I just pushed https://github.com/bigbluebutton/bigbluebutton/pull/15489/commits/e15c1711ea3417f72087fec7dbef193a527fc869 into this PR and am merging. Thanks @BrentBaccala and @GuiLeme

**Comment by paulmenzel** [comment]
Sorry, a little bit off-topic, but how can this merge/pull request be reviewed? There are 127 commits shown and 80 files changed.

**Comment by antobinary** [comment]
@paulmenzel that is my fault - I pushed a commit into this branch and had not noticed I was carrying extra commits. Let me see if I can do anything now to keep the actual changes

**Comment by paulmenzel** [comment]
@antobinary, thank you. Stuff happens, so no worries. Just wanted to make sure, I didn’t miss anything. Interested people can use the git command line.

**Comment by antobinary** [comment]
Brent opened a pull request with only https://github.com/BrentBaccala/bigbluebutton/commit/29adba9a030c1293bb2f8bf786c083710ed6f31f
@GuiLeme suggested https://github.com/BrentBaccala/bigbluebutton/commit/21b08db177c2c2f1f5cab8e5b6c7fe426ceb7e7a and I included it
And then I pushed https://github.com/BrentBaccala/bigbluebutton/commit/e15c1711ea3417f72087fec7dbef193a527fc869 into it too ..
So altogether it forms... https://github.com/bigbluebutton/bigbluebutton/compare/v2.6.0-alpha.2...antobinary:total-tomcat-changes?expand=1

**Pull request merged**